internal/datatransfer/s3/sync.go

```diff
@@ -5,18 +5,18 @@ import (
 	"crypto/x509"
 	"encoding/base64"
 	"fmt"
+	"net/http"
```

Comment:
Why do we need this import?

Reply:
this is just re-organizing the imports. not new addition (moved from line 14)

Reply:
ah, I missed it.

Reply:
@pkliczewski @masayag vscode's GO extension uses all kind of GO tools (e.g. gopls, golint) and they do it automatically.